projects/plugins/crm/admin/dashboard/main.page.php

```diff
@@ -72,7 +72,7 @@ function jpcrm_render_dashboard_page() {
 			'count'          => $count,
 			'backfill_count' => $backfill_count,
 			'contact_status' => $contact_status,
-			'link'           => false,
+			'link'           => jpcrm_esc_link( $zbs->slugs['managecontacts'] . '&quickfilters=status_' . $contact_status ),
```

Comment:
I think this got mixed up with another branch.

Reply:
Indeed; not sure what happened. Repushed with just the relevant commits.